saleor/graphql/webhook/types.py

```diff
@@ -30,6 +85,12 @@ class Webhook(CountableDjangoObjectType):
         required=True,
     )
     app = graphene.Field("saleor.graphql.app.types.App", required=True)
+    deliveries = FilterInputConnectionField(
```

Comment:
For this field we should also add a resolver and a dataloader.

Reply:
Just a comment: after discussion we will not add dataloaders to FilterInputConnectionField fields.